test/README.md

```diff
@@ -0,0 +1,12 @@
+# Test directory
+
+This is default directory for salgenerator usage.
+If you wish to use another location, it can be set using the
+environment variable SAL\_WORK\_DIR.
+
+If you wish to generate SAL language bindings and libraries in some other directory, you should:
```

Comment:
Please consider "should (within the desired work directory):"

Reply:
test/README.md was deleted.